es-ds-styles/scss/_modal.scss

```diff
@@ -112,13 +112,15 @@
   position: relative;
   @include border-radius.border-bottom-radius(variables.$modal-content-inner-border-radius);
 
-  &::before {
-    border-top: variables.$modal-footer-border-width solid variables.$modal-footer-border-color;
-    content: '';
-    left: 0;
-    position: absolute;
-    right: 0;
-    top: variables.$modal-footer-padding-bottom;
+  &.footer-separator {
+    &::before {
```

Comment:
Yeah I think it's fair to move the `padding-top: variables.$modal-footer-padding-top;` above into the `&.footer-separator` class, and have the default up top be `padding-top: 1rem`.